### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kendallstrautman**
Adds some more setup steps based on some friction points when cloning this down and running it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,47 @@
...
+> Refer to the [User Management](https://workos.com/docs/user-management) documentation for reference.
+
 ## Project setup
 
 To get started, clone the repo:
 
 ```bash
-git clone https://github.com/workos/next-authkit-example.git
+
+git clone git@github.com:workos/next-authkit-example.git
```
Comment (kendallstrautman) [inline README.md:13]: I think cloning with SSH should be preferred.

File: src/app/callback/route.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ export async function GET(request: NextRequest) {
 
       return response;
     } catch (error) {
+      console.error(error);
```
Comment (kendallstrautman) [inline src/app/callback/route.ts:46]: I didn't have my `JWT_SECRET_KEY` set and I saw no error, so adding this to give more clues to folks if their setup isn't right. 
Screenshot 2024-01-16 at 8 46 45 AM

File: README.md (hunk 1 above)
Comment (andy-hook) [reply README.md:13]: We can probably omit these cloning instructions entirely tbh, they don't add much given how common of a process it is.

File: README.md [hunk 3]
```diff
@@ -2,25 +2,47 @@
...
 ```
 
 Navigate to the cloned repo and install the dependencies:
 
 ```bash
-npm install
+cd next-authkit-example && npm install
```
Comment (andy-hook) [inline README.md:19]: I like the [format](https://github.com/workos/authkit?tab=readme-ov-file#prerequisites) used in the primary AuthKit project (pre-reqs followed by numbered steps).

How'd you feel about aligning with that here?

File: src/app/callback/route.ts [hunk 4]
```diff
@@ -43,6 +43,7 @@ export async function GET(request: NextRequest) {
...
       return NextResponse.json(error);
```
Comment (andy-hook) [inline src/app/callback/route.ts:47]:
```suggestion
      return NextResponse.json({
        error: error instanceof Error ? error.message : String(error),
      });
```

Should we also update the response to correctly output an error message?

**Review by andy-hook — COMMENTED** [review]
Thanks for taking the time to improve this @kendallstrautman ! left a few comments for improvements while we're in this area.

File: src/app/callback/route.ts (hunk 4 above)
Comment (kendallstrautman) [reply src/app/callback/route.ts:47]: Love it.

File: README.md (hunk 3 above)
Comment (kendallstrautman) [reply README.md:19]: Nice, yeah I like that, will rework.

Comment (kendallstrautman) [reply README.md]: @andy-hook Reworked it, lmk your thoughts ☺️

**Review by andy-hook — APPROVED** [review]
Looks great!

**Pull request merged**